src/lib/util/Google.ts

```diff
@@ -0,0 +1,75 @@
+import { KlasaClient, KlasaMessage } from 'klasa';
+import { TOKENS } from '../../../config';
+import { Events } from '../types/Enums';
+import { fetch, FetchResultTypes } from './util';
+
+const GOOGLE_MAPS_API_URL = 'https://maps.googleapis.com/maps/api/geocode/json';
+
+export async function queryGoogleMapsAPI(message: KlasaMessage, client: KlasaClient, location: string) {
+	const url = new URL(GOOGLE_MAPS_API_URL);
+	url.searchParams.append('address', location);
+	url.searchParams.append('key', TOKENS.GOOGLE_MAPS_API_KEY);
+	const { results, status } = await fetch(url, FetchResultTypes.JSON) as GoogleMapsResultOk;
+
+	if (status !== 'OK') throw message.language.tget(handleNotOK(status, client));
+	if (results.length === 0) throw message.language.tget('GOOGLE_ERROR_ZERO_RESULTS');
+
+	return [
+		results[0].formatted_address,
+		results[0].geometry.location.lat,
+		results[0].geometry.location.lng,
+		results[0].address_components
+	] as [string, number, number, GoogleMapsOkAddressComponent[]];
```

Comment:
This should return an object for readability.

Reply:
fixed in 42548da